zds/tutorialv2/views/validations_contents.py

```diff
@@ -596,23 +597,86 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
-class MarkObsolete(LoginRequiredMixin, PermissionRequiredMixin, FormView):
-    permission_required = "tutorialv2.change_validation"
+class DecideObsolete(LoginRequiredMixin, PermissionRequiredMixin, FormView):
+    permission_required = "tutorialv2.change_publishablecontent"
+
+    def get(self, request, *args, **kwargs):
+        raise Http404("Décider si un contenu est obsolète n'est pas disponible en GET.")
```

Comment:
```suggestion
    http_method_names = ["post"]
```

cf par exemple https://github.com/zestedesavoir/zds-site/blob/825b347f921ee906ec7bb19bb1d3b5419adecef2/zds/tutorialv2/views/labels.py#L53